C-Plus-Plus/math/README.md

```diff
@@ -1 +0,0 @@
-a
```

Comment:
What explains this change? It is redundant.

Reply:
yes this is redundant. It was by mistake..

Reply:
Revert this change then.